packages/rich-text/src/get-last-child-index.js

```diff
@@ -12,8 +12,8 @@ import { LINE_SEPARATOR } from './special-characters';
  *
  * @return {Array} The index of the last child.
  */
-export function getLastChildIndex( { text, formats }, lineIndex ) {
-	const lineFormats = formats[ lineIndex ] || [];
+export function getLastChildIndex( { text, replacements }, lineIndex ) {
+	const lineFormats = replacements[ lineIndex ] || [];
```

Comment:
Variable should be renamed to reflect the fact it doesn't consume from `formats` ?

(Same note below for `formatsAtIndex`)

Reply:
Yeah, we could rename it. I'm not 100% sure on the naming tbh. What do we call these arrays holding information about a certain index? I left it as `formats` because in my head I still think about it as regular formats, line formats, and object formats. I do agree that this could benefit from better naming.

Reply:
I'm going to leave this one for now as I'm not sure about the naming. Until now I saw a format as the equivalent of an element. So in that sense, the naming is still correct.

Reply:
> I left it as `formats` because in my head I still think about it as regular formats, line formats, and object formats. I do agree that this could benefit from better naming.

I'd wondered if it'd just be as simple as applying the new "replacement" nomenclature universally.

e.g. `lineReplacements` or `replacementsAtIndex`